primitives/src/chain/ethereum/receipt.rs

```diff
@@ -3,13 +3,12 @@ use crate::{
     bytes,
     chain::ethereum::{EthereumHeader, EthereumHeaderJson, MMRProof, MMRProofJson},
     hex,
+    array::{Bloom, H256, H160},
```

Comment:
why use self-defined hash type?

Reply:
because parity-common has not published it's latest version to the rust hub.
the older version of the types cannot be used by decoder.